src/components/pages/strategies/Active/ActiveTable.tsx

```diff
@@ -79,24 +77,18 @@ export const ActiveTable = memo(function ActiveTable({
               <TableRow key={strategy.transactionHash}>
                 <TableCell>{strategy.created.toLocaleString()}</TableCell>
                 <TableCell>
-                  {strategy.quoteTokenDetails && strategy.baseTokenDetails
-                    ? `${strategy.quoteTokenDetails?.symbol} - ${strategy.baseTokenDetails?.symbol}`
+                  {strategy.quoteToken && strategy.baseToken
+                    ? `${strategy.quoteToken?.symbol} - ${strategy.baseToken?.symbol}`
                     : "Unknown"}
                 </TableCell>
                 <TableCell>{strategy.brackets.length}</TableCell>
                 <TableCell>
-                  {formatSmart(
-                    strategy.totalBaseBalance(),
-                    strategy.baseTokenDetails?.decimals || 18
-                  ) || "-"}{" "}
-                  {strategy.baseTokenDetails?.symbol}
+                  {strategy.baseFunding.toSD(4).toString() || "-"}{" "}
```

Comment:
`toSD` method got me confused. It doesn't seem to do what I think it does: https://mikemcl.github.io/decimal.js/#toSD

Take a look at their example:

```
x = new Decimal(9876.54321)

x.toSignificantDigits()                          // '9876.5'
x.toSignificantDigits(6)                         // '9876.54'
x.toSignificantDigits(6, Decimal.ROUND_UP)       // '9876.55'
x.toSD(2)                                        // '9900'
x.toSD(2, 1)                                     // '9800'
x                                                // '9876.54321'
```

My understanding is that `toSD` will make anything over 9999 to lose data, turning 19991.toSD(4) into 19990 and so on.

Reply:
Just got and example. Check the balance on top and the bracket balance:

![screenshot_2020-11-18_11-09-16](https://user-images.githubusercontent.com/43217/99576145-716d1080-298e-11eb-84f2-98b661f17395.png)

Reply:
Oh crap, that's a very good catch. I completely understood that wrong. Thank you so much, I'll fix that before I move anything from this codebase

Reply:
Forgot to mention yesterday, I fixed it on https://github.com/gnosis/safe-cmm-app-react/pull/160